### PR TITLE
[14.0][FIX] product_pricelist_assortment: change id of an access rule

### PR DESCRIPTION
**Pull request by ilyasProgrammer**
otherwise new ir.model.access record will not be applied after update on existing db

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@pedrobaeza fix after #1362

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

**Comment by pedrobaeza** [comment]
Why they are noupdate=1? These rules are done with CSV noupdate=0, no with XML.

**Comment by pedrobaeza** [comment]
And why not using the usual CSV form (`ir.model.access.csv`)?

**Comment by francesco-ooops** [comment]
@pedrobaeza could this work?

File: product_pricelist_assortment/security/product_pricelist_assortment_item.xml [hunk 1]
```diff
@@ -1,25 +1,7 @@
...
-    <record model="ir.model.access" id="product_pricelist_assortment_item_manager">
-        <field name="name">product.pricelist.assortment pricelist manager</field>
-        <field name="model_id" ref="model_product_pricelist_assortment_item" />
-        <field name="group_id" ref="sales_team.group_sale_manager" />
-        <field name="perm_read" eval="1" />
-        <field name="perm_create" eval="1" />
-        <field name="perm_write" eval="1" />
-        <field name="perm_unlink" eval="1" />
-    </record>
+<odoo noupdate="0">
```
Comment (pedrobaeza) [inline product_pricelist_assortment/security/product_pricelist_assortment_item.xml]: This can be noupdate=1 as previously

**Comment by pedrobaeza** [comment]
Note that on existing installations, you have the noupdate flag switched.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge minor

**Pull request merged**